app/views/waste_exemptions_engine/pdfs/certificate.html.erb

```diff
@@ -1,192 +1,189 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8">
-    <title></title>
+    <meta charset="utf-8" />
+
+    <%= wicked_pdf_stylesheet_link_tag "pdfs/govuk-frontend-2.10.0.min.css" -%>
+
     <style type='text/css'>
       * {
         font-family: Helvetica, Arial, Sans-Serif !important;
         background: #FFF;
       }
-      h1 {
-        margin-bottom: 20px;
-      }
       body {
         margin: 40px 60px;
       }
-      table tr td:first-child {
-        width: 220px;
+      .table-wrapper {
+        border-left: 5px solid #005ea5;
+        padding-left: 15px;
       }
-      ul, li {
-        list-style-type: none;
-        margin-left: 0;
-        padding-left: 0;
-      }
-      li, td, p {
-        font-size: 18px;
-        line-height: 1.5;
-      }
-      table th {
-        margin-top: 20px;
-        text-align: left;
-        font-size: 22px;
+      .govuk-table__header {
+        width: 350px;
+        vertical-align: top;
       }
     </style>
   </head>
+
   <body>
-    <div id="wrapper">
-      <main id="content" role="main" class="group">
-        <div class="grid-wrapper">
-        <div class="grid-2-3">
+    <div class="govuk-width-container">
+      <main class="govuk-main-wrapper">
+        <div class="govuk-grid-row">
+          <div class="govuk-grid-column-two-thirds">
+            <h1 class="govuk-heading-l"><%=t ".confirmation.heading", registration_number: presenter.reference %></h1>
+            <p class="govuk-body">
+              <%=t ".confirmation.paragraph1" %>
+            </p>
 
-        <!-- Header section -->
-        <header class="page-header group">
-          <div class="hgroup">
-            <h1 class="heading-small"><%=t ".heading" %></h1>
-          </div>
-        </header>
+            <p class="govuk-body">
+              <strong> <%=t ".confirmation.warning" %> </strong>
```

Comment:
```suggestion
              <strong><%=t ".confirmation.warning" %></strong>
```

Maybe a nitpick, but can we remove this extra spacing? There are a few other tags with this spacing on the page (mostly `td`s) but it doesn't look consistent.